### DISCUSSION TITLE
Replica  problem  after  alidbsdev03  /  is full.

### DISCUSSION THREAD

**Discussion by jetchan88** [autobase-tech/autobase#622] [Q&A] [answered]
root@alidbsdev02:/data/psqltbsp# patronictl list
+ Cluster: postgres-cluster (7307507502278680192) -+----+-----------+
| Member      | Host           | Role    | State   | TL | Lag in MB |
+-------------+----------------+---------+---------+----+-----------+
| alidbsdev01 | 192.168.172.68 | Replica | stopped |    |   unknown |
| alidbsdev02 | 192.168.172.69 | Replica | stopped |    |   unknown |
| alidbsdev03 | 192.168.172.70 | Leader  | running | 40 |           |
+-------------+----------------+---------+---------+----+-----------+

Even I ran  patronictl reinit postgres-cluster

**Comment by vitabaks** [comment, marked as answer]
Please attach Patroni log from replica

`journalctl -u patroni -n 100`
